src/processors/default/processMempool.ts

```diff
@@ -0,0 +1,64 @@
+
+
+import _ from 'lodash';
+
+import { Table } from '../../db/db';
+import { pendingMempoolMessages } from '../../triggers/messages';
+import { CrdtOpetation, CrdtState, MempoolMessage, PendingMempoolMessage } from '../../types/message';
+import { Clients, Processor } from '../../types/processor';
+
+export const processMempool: (table: Table) => Processor = 
+  (table) => ({
+
+    name: 'processMempool',
+    trigger: pendingMempoolMessages(table),
+    processorFunction: async (messages: PendingMempoolMessage[] , clients: Clients) => {
+      // ignore inserts for now
+      const staleMessages: MempoolMessage[] = messages.filter(message => message.operation === CrdtOpetation.INSERT);
+
+      // messages are ordered by timestamp.
+      //  group by table, column, entity so that we can easily categorize fresh and stale messages
+      const groupedMessagesUpdates = _.groupBy(
+        messages.filter(message => message.operation === CrdtOpetation.UPDATE),
+        message => `${message.operation}.${message.table}.${message.column}.${message.entityId}`
+      );
+      const entityUpdates: { [id: string]: { [column: string]: string } } = {};
```

Comment:
hmm this is primarily working for string types? maybe will need to adjust types/have a slightly more complex type system to work for columns of other types too?

Reply:
i guess could be complex since we are storing the update as a string in the db already, so we do kinda need to serialize all updates into strings for this table and then deserialize them into the right type for the objectUpdates. still todo to think about best way to handle this

Reply:
could potentially just make this CRDT string-focused only and so not support other types. though if we go that route, would be nice to have some validation/constraint on messages that they only allow columns which reference string types.

Reply:
It should work with any field that can be serialized into a string. I've tested it with string, bigint, dates, json and booleans which should cover all the types used in our db